model.py

```diff
@@ -98,7 +98,8 @@ def challenger_full_name(self):
         return f'{self.challenger_title or ""} {self.challenger_name}'.strip()
 
     def __str__(self):
-        return f"{self.perf_name} {self.mode()} challenge from {self.challenger_full_name()}({self.challenger_rating}) ({self.id})"
+        return f"{self.perf_name} {self.mode()} challenge from {self.challenger_full_name()}({self.challenger_rating})" \
+               f" ({self.id})"
```

Comment:
Putting parentheses around the multi-line string is less fragile than escaping the newline. Accidentally putting space characters after the backslash will cause errors well after the program starts.